Project-II-Calculator/calculator/src/components/ButtonComponents/NumberButton.js

```diff
@@ -1,2 +1,51 @@
 import React from 'react';
 import './Button.css';
+
+const rowThree = [7, 8, 9, 'X', 4, 5, 6, '-', 1, 2, 3, '+'];
+
+// const rowFour = [];
+
+// const rowFive = [];
+
+const NumberButton = () => {
+    return (
+    <div className="num_row">
+        <div className="keys">7</div>
```

Comment:
I'd recommend using map to render out the divs and just pass the classnames and text on props